apps/system-e2e/src/tests/islandis/application-system/acceptance/parental-leave.spec.ts

```diff
@@ -16,13 +20,51 @@ import {
   EmailAccount,
   makeEmailAccount,
 } from '../../../../support/email-account'
-import { helpers, locatorByRole } from '../../../../support/locator-helpers'
+import { helpers } from '../../../../support/locator-helpers'
 import { session } from '../../../../support/session'
-import { mockApi } from '../../../../support/api-tools'
 import { setupXroadMocks } from './setup-xroad.mocks'
 
 test.use({ baseURL: urls.islandisBaseUrl })
 
+async function getEmployerEmailAndApprove(employer: EmailAccount, page: Page) {
+  const { proceed } = helpers(page)
+
+  const email = await employer.getLastEmail(6)
+
+  if (email && typeof email.html === 'string') {
+    const employerUrlMatch = email.html.match(/>(http?:.*)<\/p>/)
+    if (employerUrlMatch?.length != 2)
```

Comment:
Do you _need_ 2 urls?

Reply:
No the match() method returns an array that contains two urls, the first one contains the `>` and `</p>` but the second one is just the url
```
employerUrlMatch:  [
  '>http://localhost:4242/umsoknir/tengjast-umsokn?token=xxx</p>',
  'http://localhost:4242/umsoknir/tengjast-umsokn?token=xxx',
  index: 42367,
  input: '',
  groups: undefined
]
```